src/controllers/auth.controller.js

```diff
@@ -21,11 +21,18 @@ const authenticate = async (user) => {
     }
 }
 
-// const authorize = async (password, hash) => {
-//     // use bcrypt to properly compare user plaintext pw to hash
-// }
+const authorize = async (password, hash) => {
+    // use bcrypt to compare user plaintext pw to hash
+    const match = await bcrypt.compare(password, hash);
 
-// exports.authorize = authorize;
+    if (match) {
```

Comment:
Make the variable name `isMatch` and just return isMatch.

Reply:
Good point, was thinking there was something less clumsy I could do there